akka-projection-cassandra/src/it/scala/akka/projection/cassandra/CassandraProjectionSpec.scala

```diff
@@ -206,6 +206,13 @@ class CassandraProjectionSpec
 
   private def concatHandlerFail4(): ConcatHandlerFail4 = new ConcatHandlerFail4
 
+  def offsetShouldBe(projectionId: ProjectionId, expectedOffset: Long) = {
+    eventually {
+      val currentOffset = offsetStore.readOffset[Long](projectionId).futureValue
+      currentOffset shouldBe Some(expectedOffset)
+    }
+  }
```

Comment:
cassandra use atLeastOnce so we should always use `eventually` when checking the offset.